src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+  background-color: white;
+  padding-inline: 16px;
+
+  &__image {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 32px 3px 0;
+  }
+
+  &__title {
+    margin-top: 40px;
+    color: $Main-accent;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px; /* 150% */
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments) Fix all cases

Reply:
Still not fixed